core/src/main/java/io/questdb/std/datetime/microtime/Timestamps.java

```diff
@@ -143,6 +147,14 @@ public static long addSeconds(long micros, int seconds) {
         return micros + seconds * SECOND_MICROS;
     }
 
+    public static long addMillis(long micros, int millis) {
+        return micros + millis * MILLI_MICROS;
+    }
+
+    public static long addMicros(long micros, int more_micros) {
```

Comment:
nit: in Java the convention is to use camelCase instead of snake_case for variable names:
```suggestion
    public static long addMicros(long micros, int moreMicros) {
```